Hummingbird.docc/index.md

```diff
@@ -55,9 +51,14 @@ Hummingbird is designed to require the least number of dependencies possible, bu
 - <doc:Sessions>
 - <doc:OneTimePasswords>
 
-### Tutorials
+### Hummingbird Extensions
 
-- <doc:Todos>
+Hummingbird is designed to require the least number of dependencies possible, but this means many features are unavailable to the core libraries. Additional features are provided through extensions. The Hummingbird repository comes with additional modules
+
+- ``HummingbirdJobs``
+- ``HummingbirdTLS``
+- ``HummingbirdHTTP2``
+- ``HummingbirdTesting``
```

Comment:
Building documentation brings up an error for these
```
warning: Linking to 'HummingbirdTLS' from a Topics group in 'doc://com.opticalaberration.hummingbird/documentation/index' isn't allowed
```

Reply:
I'll replicate these updates into the Hummingbird directory, and see about resolving the error - I didn't notice it yesterday when I shifted that content down from the overview